moveit_core/collision_detection_fcl/src/collision_world_fcl.cpp

```diff
@@ -52,9 +52,11 @@
 
 #include <boost/bind.hpp>
 
+rclcpp::Logger LOGGER_COLLISION_WORLD = rclcpp::get_logger("collision_world.fcl");
```

Comment:
move inside the namespace

Reply:
https://github.com/ros-planning/moveit2/pull/41/commits/a3011f83f7a76ca3856b5e8b490f587b1af43b7d